lib/Test/Nginx/Socket.pm

```diff
@@ -3309,7 +3341,7 @@ Below is an example from ngx_headers_more module's test suite:
     --- response_headers
     ! X-Foo
     --- response_body
-    x-foo: 
+    x-foo:
```

Comment:
I'm afraid the line trailing space here is significant. Otherwise this test block won't pass.